src/test/jtx/impl/Env.cpp

```diff
@@ -537,6 +537,14 @@ Env::do_rpc(std::vector<std::string> const& args)
     return response;
 }
 
+void
+Env::enableFeature(uint256 const feature)
```

Comment:
I think the use model is different for these operations.  What I'm seeing is that `enableFeature()` is intended for enabling an amendment mid-test, which is something I don't think we've done before.

That said, I agree that coordinating these tests with pull request 2108 sounds smart because it may turn your feature handling logic on it's ear.